wire-runtime-swift/src/main/swift/ProtoCodable/ProtoReader.swift

```diff
@@ -229,14 +229,14 @@ public final class ProtoReader {
             )
         }
         
-        //nil out unknownFields to avoid memory leaks caused by
+        //clear out unknownFields to avoid memory leaks caused by
         //UnsafeMutablePointer's inability to deallocate non-trivial types.
         //This is caused by unknownFields type, WriteBuffer, being a class/reference type.
         defer {
-            messageStack[token].unknownFields = nil
+            messageStack[token].unknownFields.removeAll()
```

Comment:
@lickel does the comment still apply as is?

Reply:
Probably?